composer.json

```diff
@@ -19,14 +29,21 @@
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
```

Comment:
This plugin has a couple of gotcha bugs in 0.4.0 to 0.4.2, so I'd be more specific here and state `^0.4.3`. 

`0.4.3` or later almost certainly what will typically be installed, but it's one less potential source of issue to deal with.

Reply:
@GaryJones It's a carry-over from your last PR.  I'm updating it now.  Pushing shortly.

Reply:
If we can't find fault in our own code, then we'll have transcended to a higher level of coding ability :-)